### ISSUE TITLE
User-Reported Issue: Unable to open US Pub. Participation Playbook w/IE 9.0 (Windows 7)

### ISSUE THREAD

**Issue by leilis** [opengovfoundation/madison#477]
11/26, 8:45am

From Frank Ning
Senior Policy Analyst, Health Canada
erning.han@hc-sc.gc.ca
via UserVoice 

Hello,

I am a Canadian federal employee. I am very interested in this platform. I signed up and tried to open a file (US Public Participation Playbook) but without success. I wonder if there is limitation on access since I am not a US citizen? Please confirm. Thanks.

// I asked him if he has encountered the same issue in other browsers.

**Comment by krues8dr** [comment]
Fixed with #893
